api/MouseScrollEvent.json

```diff
@@ -0,0 +1,55 @@
+{
+  "api": {
+    "MouseScrollEvent": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/MouseScrollEvent",
+        "support": {
+          "webview_android": {
+            "version_added": false
+          },
+          "chrome": {
+            "version_added": false
+          },
+          "chrome_android": {
+            "version_added": false
+          },
+          "edge": {
+            "version_added": false
+          },
+          "edge_mobile": {
+            "version_added": false
+          },
+          "firefox": {
+            "version_added": "3.5"
+          },
+          "firefox_android": {
+            "version_added": "4"
```

Comment:
On the MDN page, Firefox Mobile says "1", is Firefox Mobile different from Firefox Android?

https://developer.mozilla.org/en-US/docs/Web/API/MouseScrollEvent#Browser_compatibility

Reply:
Excellent question! Read about why that is so [here](https://github.com/mdn/browser-compat-data/pull/1715).
TL;DR when you see `firefox_android` of 1 to 3, write 4; and when 11 to 13, write 14.